packages/cojson/src/coValueState.ts

```diff
@@ -264,13 +260,27 @@ async function loadCoValueFromPeers(
   peers: PeerState[],
 ) {
   for (const peer of peers) {
+    if (peer.closed) {
+      continue;
+    }
+
+    const timeout = setTimeout(() => {
+      if (coValueEntry.state.type === "loading") {
+        console.error("Failed to load coValue from peer", peer.id);
+        coValueEntry.dispatch({
+          type: "not-found-in-peer",
+          peerId: peer.id,
+        });
+      }
+    }, 3000);
+
     if (coValueEntry.state.type === "available") {
-      await peer.pushOutgoingMessage({
+      void peer.pushOutgoingMessage({
         action: "load",
         ...coValueEntry.state.coValue.knownState(),
       });
```

Comment:
We don't need to wait for the message to be delivered here.

This way when the coValue becomes available because it's cached we don't wait for the websocket peer to consume the messages queue before moving forward.